libqtile/core/manager.py

```diff
@@ -361,6 +361,35 @@ def _process_screens(self) -> None:
             )
             self.screens.append(scr)
 
+    def reconfigure_screens(self, ev=None):
+        """
+        This can be used to set up screens again during run time. Intended usage is to
+        be called when the screen_change hook is fired, responding to changes in
+        physical monitor setup by configuring qtile.screens accordingly. The ev kwarg is
+        ignored; it is here in case this function is hooked directly to screen_change.
+        """
+        screens = []
+        screen_info = self.core.get_screen_info()
+        logger.info("Reconfiguring screens to: {}".format(screen_info))
+
+        for i, (x, y, w, h) in enumerate(screen_info):
+            if i < len(self.screens):
+                scr = self.screens[i]
+            elif i < len(self.config.screens):
+                scr = self.config.screens[i]
+            else:
+                scr = Screen()
+
+            scr._configure(self, i, x, y, w, h, self.groups[i])
+            scr.group.layout_all()
+            screens.append(scr)
```

Comment:
Can we use `_process_screen` here instead of repeating the code?

Reply:
I believe so! Just pushed.